common/lib/api-client/models.js

```diff
@@ -177,4 +177,19 @@ module.exports = {
       filesize: '',
     },
   },
+  allocation: {
+    attributes: {
+      from_location_id: '',
```

Comment:
Are these not a relationship like they are on `move`?

Reply:
Good point 👍